nekoyume/Assets/_Scripts/Game/TableSheets.cs

```diff
@@ -173,7 +174,7 @@ public TableSheets(IDictionary<string, string> sheets)
         public StakeRegularFixedRewardSheet StakeRegularFixedRewardSheet { get; private set; }
 
         public CrystalFluctuationSheet CrystalFluctuationSheet { get; private set; }
```

Comment:
```suggestion
        public StakeRegularFixedRewardSheet StakeRegularFixedRewardSheet { get; private set; }
        
        public StakePolicySheet StakePolicySheet { get; private set; }

        public CrystalFluctuationSheet CrystalFluctuationSheet { get; private set; }
```

시트 용도에 맞춰서 줄맞춤하고 끼워주시면 좋겠습니다